kalite/coachreports/hbtemplates/coach_nav/detailspanelbody.handlebars

```diff
@@ -1,37 +1,40 @@
 <div class="tabbable tabs-left">
-	<ul class="nav nav-tabs">
-		{{#each collection}}
-		<li class="detail-tab {{#if correct }}correct{{else}}incorrect{{/if}}{{#if @first}} active{{/if}}">
-			<a href="#{{ id }}" data-toggle="tab" aria-expanded="true">Question {{ math @index "+" 1 }}</a>
-		</li>
-		{{/each}}
-	</ul>
-	<div class="tab-content">
-		{{#each collection}}
-		<div class="tab-pane {{#if @first}}active{{/if}}" id="{{ id }}">
-			<p class="detail-tab {{#if correct }}correct">{{_ "Correct" }}{{else}}incorrect">{{_ "Incorrect" }}{{/if}}</p>
-			<p>{{_ "Time taken" }}: {{ math time_taken "/" 1000 }} {{_ "seconds" }}</p>
-			<table class="table table-striped table-bordered details-table">
-			<thead>
-			<tr>
-				<th>{{_ "Attempt" }}</th>
-				<th>{{_ "Action" }}</th>
-				<th>{{_ "Time" }}</th>
-				<th>{{_ "Status" }}</th>
-			</tr>
-			</thead>
-			<tbody>
-			{{#each response_log }}
-			<tr>
-				<td>{{ math @index "+" 1  }}</td>
-				<td>{{ type }}{{#if answer}}: {{answer}}{{/if}}</td>
-				<td>{{ datetime timestamp }}</td>
-				<td>{{#ifcond correct "===" true}}{{_ "Correct" }}{{else}}{{#ifcond correct "===" false }}{{_ "Incorrect" }}{{/ifcond}}{{/ifcond}}</td>
-			</tr>
-			{{/each}}
-			</tbody>
-			</table>
-		</div>
-		{{/each}}
-	</div>
+    <ul class="nav nav-tabs">
+        {{#each collection}}
+        <li role="presentation" class="detail-tab {{#if correct }}correct{{else}}incorrect{{/if}}{{#if @first}} active{{/if}}">
+            <a href="#{{ id }}" data-toggle="tab" aria-expanded="true">Question {{ math @index "+" ../start_number }}</a>
```

Comment:
Important change on this line, to have Question numbers scale correctly with page....